src/App.jsx

```diff
@@ -14,57 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedList, setSelectedList] = useState('Jam');
```

Comment:
The name is a bit confusing, as it's not a list, but single item. Consider naming it `selectedGood` instead